packages/ui/resolution/tests/resolution-specs.js

```diff
@@ -11,27 +12,207 @@ describe('Resolution Specs', () => {
 		{name: 'vga', pxPerRem: 8, width: 640, height: 480, aspectRatioName: 'standard'},
 		{name: 'xga', pxPerRem: 16, width: 1024, height: 768, aspectRatioName: 'standard'},
 		{name: 'hd', pxPerRem: 16, width: 1280, height: 720, aspectRatioName: 'hdtv'},
-		{name: 'fhd', pxPerRem: 24, width: 1920, height: 1080, aspectRatioName: 'hdtv', base: true},
+		{name: 'fhd', pxPerRem: 24, width: 1920, height: 1080, aspectRatioName: 'hdtv'},
```

Comment:
Why do you change the base screen type?

Reply:
I changed base type to match with sandstone `screenTypes.json`.